<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>97</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="1" column="1">
    <widget class="QSlider" name="slider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QLabel" name="thresLabel">
     <property name="toolTip">
      <string>The GREEN LED attached to PD5 will toggle when the counter crosses this</string>
     </property>
     <property name="text">
      <string>SECONDS</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="thresholdSlider">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Counts</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Set Time Interval For Counting</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="numDigits">
      <number>6</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="intValue" stdset="0">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QLCDNumber" name="count">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="numDigits">
      <number>5</number>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2" colspan="2">
    <widget class="QLabel" name="stateLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>START</string>
     </property>
     <property name="clickable" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>count</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Frame</receiver>
   <slot>setTime(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setTime(int)</slot>
 </slots>
</ui>
